book/run/sync-op-mainnet.md

```diff
@@ -0,0 +1,50 @@
+# Sync OP Mainnet
+
+To sync OP mainnet, the Bedrock datadir needs to be imported to use as starting point.
+Blocks lower than the OP mainnet Bedrock fork, are built on the OVM and cannot be executed on the EVM.
+For this reason, the chain segment from genesis until Bedrock, must be manually imported to circumvent
+execution in reth's sync pipeline.
+
+Importing OP mainnet Bedrock datadir requires exported data:
+
+- Blocks [and receipts] below Bedrock
+- State snapshot at first Bedrock block
+
+## Manual Export Steps
+
+See <https://github.com/testinprod-io/op-geth/pull/1>.
+
+Output from running the command to export state, can also be downloaded from <https://datadirs.testinprod.io/world_trie_state_op_mainnet_b2c2b6e7edb919a0b856b9fd9aa02b11ead5305e63cdb33386babd82b9bc4cfe.jsonl.zst>.
+
+## Manual Import Steps
+
+### 1. Import Blocks
+
+Imports a `.rlp` file of blocks.
+
+Note! Requires running in debug mode (TODO: <https://github.com/paradigmxyz/reth/issues/7650>).
```

Comment:
why does this matter? why would this not work with the released binary?

Reply:
performance bottleneck in bodies stage, most likely due to insertion of unsorted entries